spec/cookbook_spec.rb

```diff
@@ -234,6 +234,28 @@ def repo_path
       :result => [],
       :result_with_symlink_tracking => [['cb_one', :modified]],
     },
+    {
+      :name => 'replacing metadata file with symlink',
+      :files => [
+        {
+          :status => :deleted,
```

Comment:
This doesn't have the file being added ,right? This isn't really testing the change. A metadata being deleted, by itself, with symlink tracking on should NOT result in ":modified" unless a symlink is ALSO being added and it points to a file.

Reply:
https://github.com/facebook/between-meals/blob/8be550129834a0d5de57acf26b83084fb96f7fef/spec/cookbook_spec.rb#L280-L282 patches `file?` for this metdata.rb input, which should be testing the change. Also - the symlink could have previously existed and not be part of this changeset.

Reply:
Sure, that's also a valid test. But I also want to test the case where both are done in the same changeset.